app/config/config.yml

```diff
@@ -81,6 +84,9 @@ doctrine:
                 charset:      UTF8
         types:
             datetime: Akeneo\Bundle\StorageUtilsBundle\Doctrine\DBAL\Types\UTCDateTimeType
+            native_json:
+                class: Akeneo\Bundle\StorageUtilsBundle\Doctrine\DBAL\Types\JsonType
+                commented: true
```

Comment:
@pierallard yes!
PR for EE dev is coming 

about the standard editions, I don't if we should do it now or during the next release
they currently rely on CE DEV master, whereas this "feature" is only available on my the branch TIP-613-unified of my fork